quiz/backend/config/settings.py

```diff
@@ -132,3 +127,69 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+# Logging
+
+LOGGING_CONFIG = None
+LOGLEVEL = os.environ.get('LOGGING_LVL', 'DEBUG')
+
+if not os.path.exists(os.path.join(BASE_DIR, 'logs')):
+    os.mkdir(os.path.join(BASE_DIR, 'logs'))
+
+LOGGING = {
```

Comment:
Puściłbym to do osobnego pliku, bo za miesiąc settings.py będą mieć 99999milionów linii kodu, i zaimportował tutaj. settings.logging.py